external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -153,11 +201,7 @@ private[kafka010] case class KafkaSource(
 
```

Comment:
Shouldn't it be set to the highest available offset in the streaming metadata log, not the highest available offset in kafka?